### PR TITLE
fix(curriculum): updated description for step 51 of RPG project as expected.

### LINKED ISSUE #53778: update directions for step 51 of RPG project
### Describe the Issue

The instructions say:
Create a variable called locations and set it to an empty array.

But the correct answer is:
`const locations = [];`

### Affected Page

https://www.freecodecamp.org/learn/javascript-algorithms-and-data-structures-v8/learn-basic-javascript-by-building-a-role-playing-game/step-51

### Your code

```



```

### Expected behavior

Shouldn't it say:
Create a constant called locations and set it to an empty array.

### Screenshots

_No response_

### System

- Device: [e.g. iPhone 6, Laptop]
- OS: [e.g. iOS 14, Windows 10, Ubuntu 20.04]
- Browser: [e.g. Chrome, Safari]
- Version: [e.g. 22]

### Additional context

_No response_

### PR DESCRIPTION
**Pull request by Gunjan-vyas**
…0d08093756d2.md

Updated description for step 51 of RPG project as expected.

Checklist:

- [x] I have read and followed the [contribution guidelines](https://contribute.freecodecamp.org).
- [x] I have read and followed the [how to open a pull request guide](https://contribute.freecodecamp.org/#/how-to-open-a-pull-request).
- [x] My pull request targets the `main` branch of freeCodeCamp.
- [x] I have tested these changes either locally on my machine, or GitPod.

Closes #53778

### CODE REVIEW

**Comment by jdwilkin4** [comment]
@Gunjan-vyas 

Please link the issue you are working on to your pull request
https://docs.github.com/en/issues/tracking-your-work-with-issues/linking-a-pull-request-to-an-issue

You need to fill this out
Screenshot 2024-02-17 at 7 28 32 PM

**Review by jdwilkin4 — APPROVED** [review]

**Comment by gikf** [comment]
Hey @Gunjan-vyas,

#53781 links to this PR, the _Closes_ should use the issue that this PR is supposed to close.

File: curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8a929e4260d08093756d2.md [hunk 1]
```diff
@@ -7,7 +7,9 @@ dashedName: step-51
 
 # --description--
 
-Create a variable called `locations` and set it to an empty array.
+In your role playing game, you will be able to visit different locations like the store, the cave and the town square. You will need to create a data structure that will hold the different locations.
```
Comment (Ksound22) [inline curriculum/challenges/english/15-javascript-algorithms-and-data-structures-22/learn-basic-javascript-by-building-a-role-playing-game/62a8a929e4260d08093756d2.md:10]: Just a nickpick

```suggestion
In your role playing game, you will be able to visit different locations like the **store**, the **cave**, and the **town square**. You will need to create a data structure that will hold the different locations. 
```

**Review by zairahira — APPROVED** [review]
LGTM 🎉 Thank you for contributing @Gunjan-vyas, looking forward for more contributions!

**Pull request merged**